core/ustring.cpp

```diff
@@ -4143,27 +4143,40 @@ String String::sprintf(const Array &values, bool *error) const {
 					}
 
 					double value = values[value_index];
-					String str = String::num(value, min_decimals);
+					bool is_negative = (value < 0);
+					String str = String::num(abs(value), min_decimals);
```

Comment:
This should likely have been `Math::abs()`.
Somehow CI passed on the `master` branch but the cherry-pick failed on `3.2`:
```
clang++ -o core/ustring.x11.opt.64.llvm.o -c -std=gnu++14 -Wctor-dtor-privacy -Wnon-virtual-dtor -O3 -pipe -fpie -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Werror -DTYPED_METHOD_BIND -DTOUCH_ENABLED -DALSA_ENABLED -DALSAMIDI_ENABLED -DJOYDEV_ENABLED -DX11_ENABLED -DUNIX_ENABLED -DOPENGL_ENABLED -DGLES_ENABLED -DNDEBUG -DPTRCALL_ENABLED -DGDSCRIPT_ENABLED -DMINIZIP_ENABLED -DZSTD_STATIC_LINKING_ONLY -DGLAD_ENABLED -DGLES_OVER_GL -DFREETYPE_ENABLED -DSVG_ENABLED -DHAVE_MNTENT -Ithirdparty/nanosvg -Ithirdparty/freetype/include -Ithirdparty/libpng -Ithirdparty/glad -Ithirdparty/zstd -Ithirdparty/zlib -Iplatform/x11 -I. core/ustring.cpp
core/ustring.cpp:4215:31: error: using integer absolute value function 'abs' when argument is of floating point type [-Werror,-Wabsolute-value]
                                        String str = String::num(abs(value), min_decimals);
                                                                 ^
core/ustring.cpp:4215:31: note: use function 'std::abs' instead
                                        String str = String::num(abs(value), min_decimals);
                                                                 ^~~
                                                                 std::abs
```

Reply:
Or `ABS()` I guess:
```
core/typedefs.h
91:#define ABS(m_v) (((m_v) < 0) ? (-(m_v)) : (m_v))
```

Reply:
Fixed up in cbcc0eacd5cd7af661ec99dcdc8e5a69a8a7ca5d (cherrypick: 362193db6a).